webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/KnoraProjectService.scala

```diff
@@ -75,6 +77,8 @@ final case class KnoraProjectService(knoraProjectRepo: KnoraProjectRepo) {
         DuplicateValueException(s"Project with the shortname: '${shortname.value}' already exists"),
       )
 
+  def erase(project: KnoraProject): Task[Unit] = knoraProjectRepo.delete(project)
```

Comment:
comment: here the naming gets a bit weird... I suppose the KnoraProjectRepo follows CRUD terminology and therefore uses delete instead of erase?

Reply:
Yes, delete on the repository level really removes the data from the database. We do not expose 'delete' methods on the service level (yet), this is done with update/change requests.

Naming is hard and if you have a better idea let me know. For me it feels awkward if we would have to name the methods on the repo level `erase…`. Also the base interface it would then be a `CrueRepository` … hardly recognizable to me.  ¯\_(ツ)_/¯

Reply:
no no, in the light of this, it's absolutely fine, this line just reads a bit awkward at first. But it really doesn't hurt, it was just a typical case of "double dip" when reading the code.